.github/actions/javascript/contributorChecklist/contributorChecklist.js

```diff
@@ -127,37 +127,58 @@ function getAllComments() {
     }, response => _.map(response.data, comment => comment.body));
 }
 
+function countCheckedBoxes(string) {
+    return string.toLowerCase().match(/\[x\]/g).length;
+}
+
+function getChecklistSectionOfPullRequestBody(pullRequestBody) {
+    const processed = pullRequestBody.match(/#### PR Author Checklist(.*)<h4>PR Reviewer Checklist<\/h4>/);
+    return processed ? processed[1] : '';
+}
+
 getPullRequestBody()
-    .then(pullRequestBody => combinedData.push(pullRequestBody))
+    .then(pullRequestBody => combinedData.push(getChecklistSectionOfPullRequestBody(pullRequestBody)))
     .then(() => getAllReviewComments())
     .then(reviewComments => combinedData.push(...reviewComments))
     .then(() => getAllComments())
     .then(comments => combinedData.push(...comments))
     .then(() => {
-        let authorChecklistComplete = false;
-        let reviewerChecklistComplete = false;
+        let authorChecklistComment = null;
+        let reviewerChecklistComment = null;
+
+        const authorChecklistNumber = countCheckedBoxes(completedAuthorChecklist);
+        const reviewerChecklistNumber = countCheckedBoxes(completedReviewerChecklist);
 
-        // Once we've gathered all the data, loop through each comment and look to see if it contains a completed checklist
+        // Once we've gathered all the data, loop through each comment and look to see if it contains enough completed checkboxes
```

Comment:
I think we can also include a word about the empty checkboxes (there should not be any)